code/modules/mob/mob.dm

```diff
@@ -802,7 +802,7 @@
 		var/mob/M = dropping
 		if(ismob(user))
 			var/mob/U = user
-			if(!iscyborg(U) || U.a_intent == INTENT_HARM)
+			if(!iscyborg(U) && !isAI(U) || U.a_intent == INTENT_HARM)
```

Comment:
```suggestion
			if((!iscyborg(U) || U.a_intent == INTENT_HARM) && !isAI(U))
```
AIs shouldn't have strip panel even on harm and besides both AI and borg are the same subtype.